File: old_odbc_connection_examples/sqldatabaseconnect.jl
using DataFrames, Query, JuliaDB

# driver/connection string found at
# https://docs.microsoft.com/en-us/sql/connect/odbc/microsoft-odbc-driver-for-sql-server?view=sql-server-ver15

#enter user information to be passed into the connection string
database= begin 
    print("Enter Database Name: ")
    readline()
end 

user= begin 
    print("Enter Username: ")
    readline()
end 


pass= begin 
    print("Enter Password: ")
    readline()
end 

# setting up database  
dsn=ODBC.DSN("Driver=[insert driver here ];Address=;Database=$databse;UID=$user;PWD=$pass;")

# select top 50 rows from sample table, select as a julia DB indexed table. 
results=ODBC.query(dsn,"select TOP 50 * from events") |> table

#writing data


ODBC.disconnect!(dsn)

File: old_odbc_connection_examples/sqldatabaseconnect.r
library(RODBC)
# tried the tidyverse ODBC package, but it kept giving me errors
library(tidyverse)
conn=odbcDriverConnect(connection="Driver=[insert driver here];Address=;Database=;UID=;PWD=C;")
dat =conn%>% sqlQuery("select * from [insert table here]")
close(conn)
